lib/improver/nbhood/recursive_filter.py

```diff
@@ -319,7 +331,21 @@ def process(self, cube, alphas_x=None, alphas_y=None, mask_cube=None):
             new_cube (iris.cube.Cube):
                 Cube containing the smoothed field after the recursive filter
                 method has been applied.
+
+        Raises:
+            ValueError: If any alpha cube value is over 0.5
         """
+
```

Comment:
(Similarly, we can halve the number of lines of code here by using a loop)

Reply:
Done